hack/lib.bash

```
@@ -161,6 +179,25 @@ kubeconfig::hostname::set() {
+kubectl::kubeconfig::secret() {
+    local kubeconfig="$1"
+    local target="$2"
+    local name="$3"
+    local hostname="$4"
+    if [[ -z "$hostname" ]]; then
+        hostname="broker-$name-control-plane:6443"
+    fi
+
+    cp "$target" "$target.tmp"
+    target="$target.tmp"
+    local cur_hostname="$(kubeconfig::hostname "$target")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)